### PR TITLE
Pinned comments on articles

### PR DESCRIPTION
**Pull request by Taniyah-L-Jackson**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [x] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by Taniyah-L-Jackson** [comment]
Adding a failsafe for the 'profile_type' and 'pinnable_type'

File: app/models/profile_pin.rb [hunk 1]
```diff
@@ -1,13 +1,14 @@
-class ProfilePin < ApplicationRecord
+class GenericPins < ApplicationRecord #migrate later
```
Comment (rhymes) [inline app/models/profile_pin.rb:1]: This won't work, I think you should keep the name for now. You can worry about the model name in a separate PR once the functionality is in.

**Review by rhymes — COMMENTED** [review]
I set the PR as draft as it seems you're still working on it. Mark it ready for review when you're done. 

Thank you @Taniyah-L-Jackson!

File: app/models/profile_pin.rb (hunk 1 above)
Comment (benhalpern) [reply app/models/profile_pin.rb:1]: Yup, this makes sense to me.

**Comment by benhalpern** [comment]
Excited about this!

File: app/models/profile_pin.rb (hunk 1 above)
Comment (Taniyah-L-Jackson) [reply app/models/profile_pin.rb:1]: I see, will do.

**Comment by citizen428** [comment]
@Taniyah-L-Jackson Hi, just checking in to see if you were planning on picking this up again or if we should close the PR?

**Comment by citizen428** [comment]
Closing in favor of #10920

**Comment by Taniyah-L-Jackson** [comment]
@citizen428 yea, it can be closed

**Pull request closed without merging**